python/libraries/websocket.py

```diff
@@ -163,7 +163,7 @@ async def server(websocket, path):
 
             if stop:
                 ws_server.close()
-
+                
```

Comment:
This is quite a powerful change

Reply:
I needed to switch branches and it would not let me until I committed